src/Analysis/Engine/Test/AnalysisTest.cs

```diff
@@ -5864,1796 +5889,1796 @@ import mod1
                     .And.HaveFunction("d").WithParameter("fn").OfType(BuiltinTypeId.Function);
             }
         }
-/*
-        [TestMethod, Priority(0)]
-        public async Task DecoratorReferences() {
-            var text = @"
-def d1(f):
-    return f
-class d2:
-    def __call__(self, f): return f
-
-@d1
-def func_d1(): pass
-@d2()
-def func_d2(): pass
-
-@d1
-class cls_d1(object): pass
-@d2()
-class cls_d2(object): pass
-";
-            var entry = ProcessText(text);
-            entry.AssertReferences("d1",
-                new VariableLocation(2, 1, VariableType.Value),
-                new VariableLocation(2, 5, VariableType.Definition),
-                new VariableLocation(7, 2, VariableType.Reference),
-                new VariableLocation(12, 2, VariableType.Reference)
-            );
-            entry.AssertReferences("d2",
-                new VariableLocation(4, 1, VariableType.Value),
-                new VariableLocation(4, 7, VariableType.Definition),
-                new VariableLocation(9, 2, VariableType.Reference),
-                new VariableLocation(14, 2, VariableType.Reference)
-            );
-            AssertUtil.ContainsExactly(entry.GetValues("f", 18).Select(v => v.MemberType), PythonMemberType.Function, PythonMemberType.Class);
-            AssertUtil.ContainsExactly(entry.GetValues("f", 66).Select(v => v.MemberType), PythonMemberType.Function, PythonMemberType.Class);
-            AssertUtil.ContainsExactly(entry.GetValues("func_d1").Select(v => v.MemberType), PythonMemberType.Function);
-            AssertUtil.ContainsExactly(entry.GetValues("func_d2").Select(v => v.MemberType), PythonMemberType.Function);
-            AssertUtil.ContainsExactly(entry.GetValues("cls_d1").Select(v => v.MemberType), PythonMemberType.Class);
-            AssertUtil.ContainsExactly(entry.GetValues("cls_d2").Select(v => v.MemberType), PythonMemberType.Class);
-        }
-
-        [TestMethod, Priority(0)]
-        public async Task DecoratorClass() {
-            var text = @"
-def dec1(C):
-    def sub_method(self): pass
-    C.sub_method = sub_method
-    return C
-
-@dec1
-class MyBaseClass1(object):
-    def base_method(self): pass
+        /*
+                [TestMethod, Priority(0)]
```

Comment:
The whole block of commented tests is offset, please undo it.

Reply:
OK. Prob formatter.